package.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"types": "./lib/index.d.ts",
 	"dependencies": {
-		"ardrive-core-js": "1.0.5",
+		"ardrive-core-js": "../ardrive-core-js",
```

Comment:
May you specify the branch to use locally at `ardrive-core-js`? The way I do is by adding an underscore then the branch name like this: `"../ardrive-core-js_<MyBranchName>"` so when I pull the `package.json` and try to build it fails. That makes the dev to read the package.json and checkout the core to the correct branch